contracts/script/deploy/verifiable/doc/DEPLOYMENT_PROCEDURE.md

```diff
@@ -0,0 +1,30 @@
+# Deployment Procedure
+
+This document details the deployment procedure using the `DeployVerifiable.sol` script. Verifying the correctness of the contracts is out of the scope of this document. The process of deploying EigenDA contracts with this script is divided into the following distinct phases:
+
+## Deployment
+
+* Use placeholder.config.json as a template to create a configuration file with all parameters that the contracts need to be initialized with. 
+* Any deployer can run the deployment script with `forge script DeployVerifiable` with the DEPLOY_CONFIG_PATH env variable pointing to the config file. Make sure to use the `--verify` flag to verify the contracts on etherscan.
+* After the script is run, a `DeploymentInitializer` contract will be deployed. The contract's address, github commit, and configuration file (if not in the commit) is to be sent over to the intended contract owner for verification.
```

Comment:
"intended contract owner" is a bit vague. Good to be more newb-proof and give an example. Or maybe we can add a section with example owners, like:
1. yourself (if this is not a very important contract, perhaps on an anvil chain)
2. some multisig (do we know exactly which one we would be using most of the time? good to embed the default in this doc if so)

Are there any others that would make sense?

Reply:
I agree that it is a bit confusing, so I did some restructuring and refer to the `initialOwner` explicitly through the doc, and add an explanation that it should be a multisig or an EOA. Stopping short of an exhaustive list since it really could be any address.

https://github.com/Layr-Labs/eigenda/pull/1408/files/5973a66bb9a01185cfe434088394df525b4c328f..f621a219a98c9e9528f89df6b30d400aac837b52